test/suites/load-balancer.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+Documentation   Load balancer test suite.
+
+Library         Process
+
+Resource        ../resources/common.resource
+Resource        ../resources/kubeconfig.resource
+
+Suite Setup     Setup Suite With Namespace
+Suite Teardown  Teardown Suite With Namespace
+
+*** Variables ***
+${USHIFT_HOST}    ${EMPTY}
+${USHIFT_USER}    ${EMPTY}
+
+*** Test Cases ***
+Load Balancer Smoke Test
+    [Documentation]    Verify that Load Balancer correctly exposes HTTP service
+    [Tags]    smoke
+    [Setup]    Run Keywords
+    ...    Create Hello MicroShift Pod    AND
+    ...    Expose Hello MicroShift Pod Via LB
+
+    Wait Until Keyword Succeeds    3x    3s    Access Hello Microshift via LB
+
+    [Teardown]    Run Keywords
+    ...    Delete Hello MicroShift Pod Route And Service
+
+*** Keywords ***
+Create Hello MicroShift Pod
+    Run With Kubeconfig    oc create -f ../e2e/tests/assets/hello-microshift.yaml -n ${NAMESPACE}
```

Comment:
This file has been renamed in main, and so has the pod it creates (used on the next line).

Maybe you should go ahead and make a copy of that asset file in ../assets as part of this PR? We're going to want to clean up that old directory at some point anyway.